src_py/cursors.py

```diff
@@ -29,6 +29,8 @@
 
 There is also a sample string cursor named 'thickarrow_strings'.
 The compile() function can convert these string cursors into cursor byte data.
+
+Alternately, you can also use Cursor objects to use colored images ingame.
```

Comment:
This kinda implies that Cursor objects are just for the surfaces, when it's really more of a unifying API.
Also, should be "in game"

Reply:
Ok thanks I will correct this

Reply:
>  You can initialize a cursor from a system cursor or use the constructor on an existing Cursor object, which will copy it. 
Providing a Surface instance will transform the cursor image into the Surface. Said Surfaces may use other colors than black and white.

Why mention 2 of the ways (+copying) to construct a cursor, but not the third?

Also "transform the cursor image into the Surface" sounds backwards to me. "Set the cursor to the image in the Surface" ?

Reply:
Oh, no, I just sucked while writing, sorry